### PR TITLE
add incubator/raw

### PR DESCRIPTION
**Pull request by taishin**
This chart was copied from the following URL because it is no longer updated.
- https://github.com/helm/charts/tree/master/incubator/raw

### CODE REVIEW

**Review by cw-sakamoto — APPROVED** [review]

File: raw/README.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+Some use cases for this chart include Helm-based installation and
+maintenance of resources of kinds:
+- LimitRange
+- PriorityClass
+- Secret
+
+## DEPRECATION NOTICE
+
+This chart is deprecated and no longer supported.
```
Comment (cw-ozaki) [inline raw/README.md:25]: @taishin 
Please remove.

File: raw/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+name: raw
+home: https://github.com/helm/charts/blob/master/incubator/raw
+version: 0.2.5
+appVersion: 0.2.3
+description: DEPRECATED A place for all the Kubernetes resources which don't already have a home.
```
Comment (cw-ozaki) [inline raw/Chart.yaml:6]: @taishin 
Please remove `DEPRECATED`.

File: raw/README.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
+This chart was copied from [helm/charts](https://github.com/helm/charts/tree/master/incubator/raw) because it is no longer updated.
+
+This software includes the work that is distributed in the [Apache License 2.0](http://www.apache.org/licenses/LICENSE-2.0).
+# ⚠️ Repo Archive Notice
+
+As of Nov 13, 2020, charts in this repo will no longer be updated.
+For more information, see the Helm Charts [Deprecation and Archive Notice](https://github.com/helm/charts#%EF%B8%8F-deprecation-and-archive-notice), and [Update](https://helm.sh/blog/charts-repo-deprecation/).
```
Comment (cw-ozaki) [inline raw/README.md:7]: @taishin 
Please remove.

File: raw/README.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+      name: app-low
+    value: 70000
+    globalDefault: false
+    description: "This priority class should only be used for low priority app pods."
+```
+
+#### STEP 2: Install your raw resources.
+
+```
+helm install --name raw-priority-classes incubator/raw -f raw-priority-classes.yaml
```
Comment (cw-ozaki) [inline raw/README.md:106]: @taishin 
Change `incubator` to `chatwork`.

File: raw/README.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+```
+
+```
+$ helm secrets enc secrets.yaml
+```
+
+#### STEP 2: Install your templated resources.
+
+```
+helm secrets install --name mysecret incubator/raw -f values.yaml -f secrets.yaml
```
Comment (cw-ozaki) [inline raw/README.md:140]: @taishin
Change `incubator` to `chatwork`.

**Review by cw-ozaki — APPROVED** [review]

**Pull request merged**